Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -224,6 +263,22 @@ extension StripeCardReaderService: DiscoveryDelegate {
 }
 
 
+// MARK: - ReaderDisplayDelegate.
+extension StripeCardReaderService: ReaderDisplayDelegate {
+    public func terminal(_ terminal: Terminal, didRequestReaderInput inputOptions: ReaderInputOptions = []) {
```

Comment:
Maybe add a comment that didRequestReaderInput means the reader wants the buyer to tap/swipe/insert a card/device (since the value is a bit hard to grok on its own)